uPortal-security/uPortal-security-mvc/src/main/java/org/apereo/portal/security/mvc/LogoutController.java

```diff
@@ -43,15 +40,19 @@
  */
 @Controller
 @RequestMapping("/Logout")
-public class LogoutController implements InitializingBean {
+public class LogoutController {
+
+    /* package-private */ static final String LOGOUT_REDIRECT_PREFIX = "logoutRedirect.";
+    /* package-private */ static final String LOGOUT_REDIRECT_ROOT = "root";
 
-    private static final Log log = LogFactory.getLog(LogoutController.class);
+    private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
You are already using `org.slf4j.Logger`.
Lombok handles the magic of handling the imports and injecting a `log` instance for you.
Wherever it is leveraged is guarantees a standard implementation.